app/services/sso/openid_connect.rb

```diff
@@ -20,6 +22,12 @@ def authenticated?
       self.user = User.current.presence || authenticate!
     end
 
+    def logout_url
+      default_logout_url = Setting['login_delegation_logout_url']
+      return unless mod_auth_openidc_session_exist?
```

Comment:
Is it important to check this? This means the method returns `nil` which may be an issue. Isn't it safe to always send the user to the logout URL?